src/layers/utils.js

```diff
@@ -16,6 +16,19 @@ export function padWithDefault(arr, defaultValue, padWidth) {
   return arr;
 }
 
+/**
+ * (Safely) get GL values for associated dtype.
+ * @param {keyof typeof import('../constants').DTYPE_VALUES} dtype
+ */
+export function getDtypeValues(dtype) {
```

Comment:
Should we export this to end-users instead of the object itself?

Reply:
A slightly different question, why do we export `DTYPE_VALUES` as a public API? All of the fields are internal constants for webgl, so only `max` could possibly be useful outside. That said, finding `min` / `max` for a given typed array is a google-away, and I'm not sure why it's a responsibility of our public API.

Reply:
Beyond its use in Vitessce, I could imagine a situation where someone might want to use our loaders in a different application that doesn't necessarily use deck.gl, but maybe a different WebGL framework.  I have been told it is very hard to find these values in a usable format to get textures to work.  If you want to de-export it, though, that is also fine.

Reply:
I think we differ in opinion here, but in general anticipating future use cases is difficult. I'd be happy to expose "features" like this if it is motivated by a tangible request (e.g. the example you describe but via an issue from a third party), but otherwise it's exposing more of the internals of Viv and things we are responsible for / cannot refactor in the future.

Reply:
Ok, that sounds good to me.